Server/Services/RevolutionaryDiscordBotService.cs

```diff
@@ -652,29 +666,37 @@ private SlashCommandBuilder BuildLanguageCommand()
                 "Displays current language translation statistics for Thrive.");
     }
 
-    private async Task HandleDsucCommand(SocketSlashCommand command)
+    private async Task HandleDaysSinceCommand(SocketSlashCommand command)
     {
         if (!await CheckCanRunAgain(command))
             return;
 
-        WatchedKeyword? underWaterCiv = await database.WatchedKeywords.FindAsync("underWaterCiv");
+        WatchedKeyword? keyword = await database.WatchedKeywords.FindAsync(
+            command.Data.Options.First().Value);
+
+        if (keyword == null)
+        {
+            logger.LogError($"Watched Keyword is null");
```

Comment:
This should not be an error as the user can put whatever they want.

Reply:
same as above

Reply:
https://github.com/Revolutionary-Games/ThriveDevCenter/pull/578#discussion_r1103865831

Reply:
This is still an error if the database call fails, which should be the only time that condition is triggered. And even if someone manages to send something that's wrong, logging it, responding with an error message, and returning is probably the best way to handle it.

Reply:
> This is still an error if the database call fails

Database call failures are exceptions. Null only means that the database successfully looked for the data but just couldn't find anything.

Anything that comes from the outside should *at most* be on the warning level. In this case with users interacting with a bot, I'd expect someone to figure out how to send bad data. That's not our fault. The logs should represent what's going on in the devcenter processing. And users sending bad data is not the devcenter's fault.